IHP/Pagination/ControllerFunctions.hs

```diff
@@ -179,12 +173,12 @@ defaultPaginationOptions =
 --
 -- *AutoRefresh:* When using 'paginatedSqlQuery' with AutoRefresh, you need to use 'trackTableRead' to let AutoRefresh know that you have accessed a certain table. Otherwise AutoRefresh will not watch table of your custom sql query.
 paginatedSqlQuery
-  :: forall model
-   . ( FromRow model
+  :: ( FromRow r
```

Comment:
Why was the type variable name changed from `model` to `r`?

Reply:
To maintain consistency with similar functions: 

https://github.com/digitallyinduced/ihp/blob/17f17b5f1bcc3475df2ba8e90f1c7dffd5ae9773/IHP/ModelSupport.hs#L379

Reply:
Makes sense, but let's keep the old name. Single variable names are not good. (We should eventually patch `sqlQuery` to fix this there as well)

Reply:
I see. That makes sense. I've updated the PR accordingly.